src/applications/edu-benefits/sass/edu-benefits.scss

```diff
@@ -42,13 +42,24 @@
 
   .search-controls {
     display: flex;
-    flex-wrap: nowrap;
-    input {
-      margin-right: 10px;
-    }
+    .search-input {
+      display: flex;
+      flex-direction: column;
+      flex-grow: 2;
+      flex-wrap: nowrap;
 
-    button {
+      .clear-search{
+        display: flex;
+        flex-direction: row;
+        justify-content: flex-end;
+        &.info {
+          justify-content: space-between;
```

Comment:
I really wanted to achieve this without class switching. Couldn't figure out how to get the Start Over link to be right justified when it was the only element in this container. 
![screen shot 2018-08-02 at 13 20 33](https://user-images.githubusercontent.com/4998130/43606083-9a37ed9a-9657-11e8-9b09-d90b4ce1c6f1.png)